cmd/apply/cmdapply.go

```diff
@@ -68,14 +72,20 @@ type ApplyRunner struct {
 	ioStreams genericclioptions.IOStreams
 	applier   *apply.Applier
 
-	output  string
-	wait    bool
-	period  time.Duration
-	timeout time.Duration
-	noPrune bool
+	output                 string
+	wait                   bool
+	period                 time.Duration
+	timeout                time.Duration
+	noPrune                bool
+	prunePropagationPolicy string
 }
 
-func (r *ApplyRunner) Run(cmd *cobra.Command, args []string) {
+func (r *ApplyRunner) RunE(cmd *cobra.Command, args []string) error {
+	prunePropPolicy, err := convertPropagationPolicy(r.prunePropagationPolicy)
```

Comment:
Should we add a comment somewhere that the default propagation policy is `background`

Reply:
We are setting the default value in the cobra command. So running `apply --help` will show that the default is `Background`. I have also updated this PR so the Options passed into the Applier will also handle defaulting to Background correctly.